cirq-google/cirq_google/devices/grid_device.py

```diff
@@ -118,11 +118,12 @@ class _GateRepresentations:
             cirq.YPowGate,
             cirq.PhasedXPowGate,
             cirq.HPowGate,
-            cirq.IdentityGate,
+            cirq.GateFamily(cirq.I),
```

Comment:
nit: Just `cirq.I` would be good, to be consistent with the other `deserialized_forms`. These are what we want to display in the gateset object, so here we keep it as simple as possible.

Reply:
cirq.I is an object (an instance of cirq.IdentityGate), the others are classes. when I use cirq.I the code breaks because it assumes classes.